src/explore-education-statistics-admin/src/pages/users/UserInvitePage.tsx

```diff
@@ -160,6 +183,31 @@ const UserInvitePage = ({
                   }}
                 />
 
+                <InviteUserPublicationRoleForm
+                  publications={model?.publications}
+                  publicationRoles={model?.resourceRoles.Publication}
+                  userPublicationRoles={form.values.userPublicationRoles}
+                  onAddUserPublicationRole={(
+                    newUserPublicationRole: InviteUserPublicationRole,
```

Comment:
Don't need an explicit type (same could be done for the `InviteUserReleaseRoleForm` above as well.